index.html

```diff
@@ -5,13 +5,28 @@
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/normalize/5.0.0/normalize.min.css">
     <link rel="stylesheet" href="main.css">
     <title>Battle Game Project</title>
+<<<<<<< HEAD
+  </head>
```

Comment:
It looks like the head is empty now. Which is good that your body html is out, but the head needs to have the CSS, title, charset, etc.